Makefile

```diff
@@ -1,7 +1,16 @@
-.PHONY: test
+.DEFAULT_GOAL := bin/ssm-env
+
+ARGS :=
+
+version := $(shell git describe --tags)
```

Comment:
Might want to limit it to just version tags in case we push others https://github.com/isobit/ndog/blob/94fe4aefeafeab1365ab3d7cfdf07d3740d2ab85/Makefile#L3